.ci/install_kata_image.sh

```diff
@@ -13,20 +13,36 @@ set -o errtrace
 cidir=$(dirname "$0")
 rust_agent_repo="github.com/kata-containers/kata-containers"
 arch=$("${cidir}"/kata-arch.sh -d)
+TEST_INITRD="${TEST_INITRD:-no}"
+AGENT_INIT="${AGENT_INIT:-no}"
 
 build_rust_image() {
 	export RUST_AGENT="yes"
 	osbuilder_path="${GOPATH}/src/${rust_agent_repo}/tools/osbuilder"
 	distro="ubuntu"
 
-	pushd "${osbuilder_path}"
-	echo "Building rust image"
-	sudo -E USE_DOCKER=1 DISTRO="${distro}" make -e image
-
 	image_path="/usr/share/kata-containers/"
 	sudo mkdir -p "${image_path}"
-	echo "Install rust image to ${image_path}"
-	sudo install -D "${osbuilder_path}/kata-containers.img" "${image_path}"
+
+	pushd "${osbuilder_path}"
+
+	if [ "${TEST_INITRD}" == "no" ]; then
+		echo "Building image with AGENT_INIT=${AGENT_INIT}"
+		sudo -E USE_DOCKER=1 DISTRO="${distro}" make -e image
+
+		echo "Install image to ${image_path}"
+		sudo install -D "${osbuilder_path}/kata-containers.img" "${image_path}"
+	else
+		echo "Building initrd with AGENT_INIT=${AGENT_INIT}"
+		export ROOTFS_DIR="${osbuilder_path}/rootfs-${distro}"
+		sudo -E AGENT_INIT="${AGENT_INIT}" USE_DOCKER=true \
+			./rootfs-builder/rootfs.sh "${distro}"
+		sudo -E ./initrd-builder/initrd_builder.sh $ROOTFS_DIR
```

Comment:
Why run rootfs and initrd builder scripts by hand instead of using 'make' like the image branch above?